templates/launch.html

```diff
@@ -153,6 +153,11 @@ <h3>Runner Data</h3>
                                         "</span>" +
                                         "</div>"
 
+                            } else if (metadataField['validator'] == "optional_string") {
```

Comment:
Instead of duplicating this whole block just add a condition to the setting of `defaultValue`